src/callbacks_stage/subcell_bounds_check.jl

```diff
@@ -138,7 +140,7 @@ end
                 continue
             end
             println(string(variables[v]) * ":\n- positivity: ",
-                    idp_bounds_delta[Symbol(string(v), "_min")][2])
+                    idp_bounds_delta_threaded[Symbol(string(v), "_min")][1][2])
```

Comment:
A comment would be nice why `[1][2]` is used here. The same holds for the other prints like this.